### PR TITLE
created basic architecture, Book model, and database connection

### PR DESCRIPTION
**Pull request by irynamekh**

### CODE REVIEW

File: src/main/java/com/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    private BigDecimal price;
```
Comment (OleksandrSaven) [inline src/main/java/com/bookstore/model/Book.java:26]: price (BigDecimal, not null)

**Review by OleksandrSaven — APPROVED** [review]
Looks good!

**Review by OleksandrSaven — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -22,6 +22,28 @@
...
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```
Comment (vovasalyha) [inline pom.xml]: Do you need this starter for this PR?

File: pom.xml [hunk 3]
```diff
@@ -22,6 +22,28 @@
...
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>test</scope>
```
Comment (vovasalyha) [inline pom.xml]: It's better to put dependencies with test scope at the end.

File: src/main/java/com/bookstore/model/Book.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "books")
```
Comment (vovasalyha) [inline src/main/java/com/bookstore/model/Book.java]: It's more common practice to use singular form of nouns for naming tables.

**Review by vovasalyha — APPROVED** [review]
GJ! Please take a look on few minor comments

**Pull request merged**